hoomd/MPIConfiguration.h

```diff
@@ -4,6 +4,7 @@
 #pragma once
 
 // ensure that HOOMDMath.h is the first thing included
```

Comment:
This comment doesn't make sense with the style guides. Is there a purpose/reason to break that convention?

Reply:
Some MPI stacks `#define` things that break headers included by `HOOMDMath.h`. If `<mpi.h>` is included first, HOOMD fails to compile on these systems.

Reply:
Might be good to reword the comment to reflect that then.